app/(platform)/social/error.tsx

```
@@ -0,0 +1,21 @@
+"use client";
+
+import { useEffect } from "react";
+
+export default function Error({ error }: { error: Error & { digest?: string } }) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)